pkgs/development/libraries/qt-5/modules/qttools.nix

```diff
@@ -5,6 +5,9 @@
   qtbase,
   qtdeclarative,
   substituteAll,
+  # clang-based c++ parser for qdoc and lupdate
+  withClang ? false,
```

Comment:
Hmm, I am not so super happy with this option name. Maybe we should name it instead withQDoc or withTools or withExtraTools? That it required clang is some detail which isn't really exposing what we want to do.

Reply:
Hmm. I would be against that because Clang also enables https://wiki.qt.io/Lupdate_clang-based_c++_parser as the comment notes… but it looks like that functionality was actually never ported to Qt5.

Does it make sense to nonetheless keep the name the same, for consistency with `qt6.qttools`? Then I'll only adjust the comment to only mention qdoc.

https://github.com/NixOS/nixpkgs/blob/cb9a96f23c491c081b38eab96d22fa958043c9fa/pkgs/development/libraries/qt-6/modules/qttools.nix#L8-L9

Reply:
I am not sure, someone please decide 😓

Reply:
Option has been removed, we always build with clang stuff added now.